docs/field-level-encryption.md

```diff
@@ -112,3 +112,41 @@ With the above connection, if you create a model named 'Test' that uses the 'tes
 const Model = mongoose.model('Test', mongoose.Schema({ name: String }));
 await Model.create({ name: 'super secret' });
 ```
+
+## Automatic FLE in Mongoose
+
+Mongoose supports the declaration of encrypted schemas - schemas that, when connected to a model, utilize MongoDB's Client Side
+Field Level Encryption or Queryable Encryption under the hood.  Mongoose automatically generates either an `encryptedFieldsMap` or a
+`schemaMap` when instantiating a MongoClient and encrypts fields on write and decrypts fields on reads.
+
+### Encryption types
+
+MongoDB has to different automatic encryption implementations: client side field level encryption (CSFLE) and queryable encryption (QE).  
+See [choosing an in-use encryption approach](https://www.mongodb.com/docs/v7.3/core/queryable-encryption/about-qe-csfle/#choosing-an-in-use-encryption-approach).
+
+###  Declaring Encrypted Schemas
+
+The following schema declares two properties, `name` and `ssn`.  `ssn` is encrypted using queryable encryption, and
+is configured for equality queries:
+
+```javascript
+const encryptedUserSchema = new Schema({ 
+  name: String,
+  ssn: { 
+    type: String, 
+    // 1
+    encrypt: { 
+      keyId: '<uuid string of key id>',
+      queries: 'equality'
+    }
+  }
+  // 2
+}, { encryptionType: 'queryable encryption' });
```

Comment:
In the kick off this is written as `encryptionType: 'qe'`, just wondering if this change was intentional? csfle is still an acronym, should we stick with qe?

Reply:
'qe' was just me being lazy.  I don't feel strongly, but if I had to choose which to put in the public API, I would choose the approach that doesn't use acronyms.  Acronyms inevitably lead to confusion

Reply:
Sure I'm fine with that but it is slightly strange to have a space in it since it is kind of an enum, thoughts?

Maybe we don't need field-level in csfle?

```js
'queryableEncryption';
'clientSideEncryption';
```